msbuild/Xamarin.Mac.Tasks.Core/Tasks/MmpTaskBase.cs

```diff
@@ -72,6 +72,10 @@ public class MmpTaskBase : ToolTask
 		public string I18n { get; set; }
 		public string ExtraArguments { get; set; }
 
+		public string AOTMode { get; set; }
+		public bool HybridAOT { get; set; }
```

Comment:
it sounds a bit confusing to to have `AOTMode` and then `HybridAOT` which is a _mode_

Maybe seeing a pic of the UI would help, but I think the terms could be clearer